native/ssz_nif/src/ssz_types/mod.rs

```diff
@@ -6,7 +6,8 @@ mod beacon_chain;
 
 type Bytes4 = [u8; 4];
 #[allow(dead_code)]
-type Bytes20 = [u8; 20];
+// type Bytes20 = [u8; 20];
```

Comment:
This can be removed. Also the `#[allow(dead_code)]` above. In fact, we should also change the `Bytes48` to `FixedVector<u8, typenum::U48>`

Reply:
Sure! I’ll update it.

Reply:
@MegaRedHand should I change `Bytes32` and `Bytes4` to their corresponding FixedVector definitions as well?

Reply:
No need, because those already implement `Encode`/`Decode`.

Reply:
Ok, I only updated Bytes48.